packages/clay-dropdown/src/ClayDropdownBase.soy

```diff
@@ -218,6 +221,10 @@
 				{sp}{$classes}
 			{/if}
 
+			{if $size}
+				{sp}btn-{$size}
```

Comment:
Hey @carloslancha the use of `btn-sm` does not seem to have any effect and does not seem to work well with `btn-unstyled`. 

Hey @pat270 could you look at this if it would be correct?

Reply:
btn-unstyled overwrites the size. It's something @pat270 might take a look, for this pr is ok

Reply:
Yes, I opened an issue for this so we can continue with this pr. #600